recipes/lrez/build.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
-make -j"${CPU_COUNT}"
+make -j"${CPU_COUNT}" LDFLAGS="${LDFLAGS} -L$( pwd )/lib"
 make install
```

Comment:
I forgot to add it to the second `make` call here. Added it now as a patch.

Reply:
(And I'm puzzled (but have not looked at it yet) why clang/ld did not complain about --whole-archive before.